colibre/config.yml

```diff
@@ -241,12 +241,12 @@ scripts:
     section: Feedback kick velocities
     title: Maximal SNII kick velocity
   - filename: scripts/stellar_abundances_FeH_OFe.py
-    caption: \\[Fe/H\\] vs \\[O/Fe\\] using Asplund et al. (2009) values for \\[Fe/H\\]Sun = 7.5 and \\[O/H\\]Sun = 8.69. The observational data for MW compiles the works of Mishenina+99, Israelian+98, Cayrel+04, Bai+04, Zhang+05, Koch+08. Most of these works assume Grevesser & Anders (1989) values for solar metallicity, their were corrected  to Asplund+09. Additional data includes Fornax (Letarte+07), Carina (Kock+05), Sculptor (Geisler+05) and  Sagittarious (Sbordone+07).
+    caption: '[Fe/H] vs [O/Fe] using Asplund et al. (2009) values for [Fe/H]Sun = 7.5 and [O/H]Sun = 8.69. The median [Fe/H] vs median [O/Fe] is indicated by the solid curve(s). The scatter points show abundancies of individual stellar particles. The observational data for MW compiles the works of Mishenina+99, Israelian+98, Cayrel+04, Bai+04, Zhang+05, Koch+08. Most of these works assume Grevesser & Anders (1989) values for solar metallicity, their were corrected  to Asplund+09. Additional data includes Fornax (Letarte+07), Carina (Kock+05), Sculptor (Geisler+05) and  Sagittarious (Sbordone+07).'
```

Comment:
This is to avoid  unwanted "\" in the captions, e.g.

![Screenshot from 2021-09-07 14-55-31](https://user-images.githubusercontent.com/20153933/132355713-d5745ff1-8ab1-4e3d-b91c-584bdc029a6d.png)